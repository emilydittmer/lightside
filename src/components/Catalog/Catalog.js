import React from 'react';
import './Catalog.scss';
import Card from '../Card/Card';

const Catalog = (props) => {
  let data = props.data.map(obj => {
    return (
      <Card 
        toggle={props.toggle} 
        type={obj.type} 
        name={obj.name} 
        data={Object.entries(obj)} 
      />
    );
  });
  
  return(
    <section className='catalog'>
      {data}
    </section>
  )
}

export default Catalog;