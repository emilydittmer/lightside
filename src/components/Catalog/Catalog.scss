@import '../../sass/_mixins.scss';
@import '../../sass/_variables.scss';

.catalog {
  grid-row: 4/5;
  background-color: hsla(0, 0%, 100%, .2);
  height: 43vh;
  border: 5px solid $main-color;

}