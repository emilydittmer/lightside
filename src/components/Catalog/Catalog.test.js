import React from "react";
import { shallow } from "enzyme";
import Catalog from "./Catalog";

describe("Catalog", () => {
  it("should match the snapshot", () => {
    const wrapper = shallow(<Catalog data={[1, 2, 3]} counter={1} />);
    expect(wrapper).toMatchSnapshot();
  });
});
