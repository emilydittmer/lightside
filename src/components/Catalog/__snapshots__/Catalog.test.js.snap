// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Catalog should match the snapshot 1`] = `ShallowWrapper {}`;
